arches/install/arches-templates/project_name/templates/index.htm

```diff
@@ -42,344 +42,313 @@
     <link href="{% static 'css/unify.css' %}" rel="stylesheet">
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.2.0/css/font-awesome.min.css">
     <link href="{% static 'css/index.css' %}" rel="stylesheet">
-
```

Comment:
I notice that the include of the accessibility.css file is missing here.  Could you please add that and ensure that this file is consistent with the main index.html file?

Reply:
@robgaston Image flexing, fixes for missing styling, added the accessibility.css include back in and ensured the install index.htm was also updated, all sorted and changes committed. Will resolve this conversation once our tester has passed the fixes.